src/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ if (WIN32)
                        ${DLL_LOCATION} ${UNIT_TEST_DIR})
 else(WIN32)
     find_package(LibXml2 2.9 REQUIRED)
-    find_package(CURL 7.31 REQUIRED)
+    find_package(CURL 7.29 REQUIRED)
```

Comment:
This is what is available in the recent CentOS 7.3.1611 release.
Of course I can do inline fix of this file (src/CMakeFiles.txt) from the %build step of ds3_c_sdk.spec